backend/index.js

```diff
@@ -3,6 +3,72 @@ const bodyParser = require('body-parser')
 const dotenv = require('dotenv');
 const MongooseConnector = require('./db-helper');
 const app = express()
+const mongoose = require('mongoose')
+const session = require("express-session");
+const passport = require("passport");
+const passportLocalMongoose = require("passport-local-mongoose");
```

Comment:
Are we using local authentication, too? What is the use of this if we aren't storing passwords?